Fingerprints/FindSamplesToCompare.wdl

```diff
@@ -0,0 +1,581 @@
+version 1.0
+import "../BenchmarkVCFs/BenchmarkVCFs.wdl" as Benchmark
+
+workflow FindSamplesToCompare {
```

Comment:
up to you, but I feel like this name sells this workflow short!  it's doing a bit more than just finding samples to compare.

Reply:
also should this go in the BenchmarkVCFs directory?  I feel like it really is a benchmarking workflow, which uses fingerprinting, as opposed to a fingerprinting workflow that also does some benchmarking on the side